### PR TITLE
Gives Warden their own Garment Bag.

### PR DESCRIPTION
**Pull request by DATA-xPUNGED**
## About The Pull Request

Puts all of warden's clothes on a garment bag, and puts said garment bag on warden's locker
## Why It's Good For The Game

Warden has almost as much alternative clothing as a head, using his locker is kind of a pain, and this oughta fix that
## Changelog
:cl:
qol: The Warden's locker now has a garment bag
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/garment.dm [hunk 1]
```diff
@@ -14,6 +14,10 @@
 	name = "head of security's garment bag"
 	desc = "A bag for storing extra clothes and shoes. This one belongs to the head of security."
 
+/obj/item/storage/bag/garment/warden
+	name = "Warden's garment bag"
```
Comment (Jolly-66) [inline code/game/objects/items/storage/garment.dm:18]: Names aren't proper for items.

```suggestion
	name = "warden's garment bag"
```

**Review by JohnFulpWillard — COMMENTED** [review]
erhm... krav maga gloves?

**Comment by DATA-xPUNGED** [comment]
> erhm... krav maga gloves?

I felt like they were more of a tool than a cosmetic, if that makes sense

**Comment by jughu** [comment]
honestly the only real alternate clothing that warden has that cant being received in other means is the blue warden jacket and the campaign hat the rest can just be received from the formal security uniforms(which has to be ordered trough cargo) metastation starts with a crate just removing the formal clothings would give the crate otself more love from cargo and auctall more reason to order it from your drip(see cmo turtleneck)

**Comment by JohnFulpWillard** [comment]
> I felt like they were more of a tool than a cosmetic, if that makes sense

We have SecHUDs/nitrile gloves in there too, I think you should include the kravs for consistency

**Comment by Jackal-boop** [comment]
hey did you actually test this. it doesnt work, atleast not on delta station

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -82,21 +82,12 @@
...
-	new /obj/item/clothing/head/hats/warden(src)
-	new /obj/item/clothing/head/hats/warden/drill(src)
-	new /obj/item/clothing/head/beret/sec/navywarden(src)
-	new /obj/item/clothing/suit/armor/vest/warden/alt(src)
-	new /obj/item/clothing/under/rank/security/warden/formal(src)
-	new /obj/item/clothing/under/rank/security/warden/skirt(src)
-	new /obj/item/clothing/glasses/hud/security/sunglasses(src)
 	new /obj/item/holosign_creator/security(src)
-	new /obj/item/clothing/mask/gas/sechailer(src)
+	new /obj/item/storage/bag/garment/warden
```
Comment (JohnFulpWillard) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:86]: Forgot to add (src) 😵‍💫

**Comment by SingingSpock** [comment]
Always. Test. Your. Own. Damn. Code.

**Comment by DATA-xPUNGED** [comment]
Shiiit

**Pull request merged**